app/models/user.rb

```diff
@@ -1,2 +1,17 @@
 class User < ApplicationRecord
+  # def recent_posts
+  #   @recent_posts_lists = []
+  #   User.all.each do |each_user|
+  #     each_user.posts.each do |each_post|
+  #      @recent_posts_lists.push(each_user.each_post.title)
+  #     end
+  #   end
+  # end
```

Comment:
- [ ] Kindly delete the commented piece of code as it makes your work look messy.